src/DocBlock/Tags/Factory/AbstractPHPStanFactory.php

```diff
@@ -44,16 +45,27 @@ class AbstractPHPStanFactory implements Factory
 
     public function __construct(PHPStanFactory ...$factories)
     {
-        $this->lexer = new Lexer(true);
-        $constParser = new ConstExprParser(true, true, ['lines' => true, 'indexes' => true]);
-        $this->parser = new PhpDocParser(
-            new TypeParser($constParser, true, ['lines' => true, 'indexes' => true]),
-            $constParser,
-            true,
-            true,
-            ['lines' => true, 'indexes' => true],
-            true
-        );
+        if (class_exists(ParserConfig::class)) {
```

Comment:
please add `class_exists` as a `use function class_exists` this will fix the codestyle issue in the pipeline.

Reply:
@jaapio Could you retrigger the build? I hope I caught everything. :)